spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/session/RedisSessionProperties.java

```diff
@@ -69,4 +74,29 @@ public void setCleanupCron(String cleanupCron) {
 		this.cleanupCron = cleanupCron;
 	}
 
+	public RedisAction getAction() {
+		return this.action;
+	}
+
+	public void setAction(RedisAction action) {
+		this.action = action;
+	}
+
+	/**
+	 * Allows specifying a strategy for configuring and validating Redis.
+	 */
+	public enum RedisAction {
```

Comment:
`ConfigurationStrategy` would be the logical name there but it fees a bit odd so a bit more brainstorming is required I think.